js/privatebin.js

```diff
@@ -851,10 +851,12 @@ jQuery.PrivateBin = (function($, RawDeflate) {
          * @param  {string} key
          * @param  {string} password
          * @param  {array}  spec cryptographic specification
+         * @param  {bool}   exportKey
          * @return {CryptoKey} derived key
          */
-        async function deriveKey(key, password, spec)
+        async function deriveKey(key, password, spec, exportKey)
         {
+            exportKey = exportKey || false;
```

Comment:
You can put that parameter in the function declarations (as in any sane language :stuck_out_tongue_winking_eye: ) since ES2015 (which we require, because of async) now, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters